com.unity.netcode.gameobjects/TestHelpers/Runtime/MockTransport.cs

```diff
@@ -19,20 +21,37 @@ private struct MessageData
         public static ulong HighTransportId = 0;
         public ulong TransportId = 0;
 
+        public float SimulatedLatencySeconds;
```

Comment:
Oh this is nice... 
I have a bunch of integration tests that need to start using this.